SCClassLibrary/Common/GUI/Base/QWebView.sc

```diff
@@ -223,9 +224,25 @@ WebView : View {
 	url { ^this.getProperty('url') }
 	url_ { |url| this.setProperty('url', url) }
 
-	enterInterpretsSelection { ^this.getProperty('enterInterpretsSelection') }
-	enterInterpretsSelection_ { |b| this.setProperty('enterInterpretsSelection', b) }
-
+	enterInterpretsSelection_ { |b|
+		enterInterpretsSelection = b;
+		if(enterInterpretsSelection){
+			this.keyDownAction = { arg view, char, mods, u, keycode, key;
+				// 01000004 is Qt's keycode for Enter, needed on Mac
+				if((char.ascii==13).or(key.asHexString == "01000004")){
+					if(mods.isShift){
+						view.runJavaScript("selectLine()",this.onInterpret(_));
+					}{
+						if(mods.isCtrl || mods.isCmd) {
+							view.runJavaScript("selectRegion()",this.onInterpret(_));
+						};
+					}
+				}
+			};
+		}{
+			this.keyDownAction = {}
+		}
```

Comment:
a couple notes here:

1. please redo the spacing on this section (see style guide here: https://github.com/supercollider/supercollider/wiki/Code-style-guidelines#whitespace), and switch from arg to pipe syntax.
2. i think `key.asHexString == "..."` could be `key == 0x...` instead, no need to convert it to a string
3. and, `char.ascii == 13` could be `char == Char.ret` or `char == $\r`.
4. the 2 innermost `if` blocks could be replaced with a `case` instead, this would be clearer in my opinion and you would have one less indentation level.